Source/FuzzySearch.swift

```diff
@@ -48,11 +68,37 @@ public struct FuzzySearchResult {
 
 public protocol FuzzySearchable {
     var fuzzyStringToMatch: String { get }
+    
+    func fuzzyTokenized() -> FuzzyTokens
+}
+
+public extension FuzzySearchable {
```

Comment:
Related to previous comment: isn't it better to leave it as implementation details and make it private?